account_creation_page.py

```diff
@@ -0,0 +1,23 @@
+import streamlit as st
+
+
+with st.form("account_creation_form"):
+
+    st.header("Social Stock Analyzer")
+    st.subheader("Create your Account")
+
+    Username = st.text_input('',placeholder='Username')
+    Firstname = st.text_input('',placeholder='Firstname')
+    Lastname = st.text_input('',placeholder='Lastname')
+    Password = st.text_input('',placeholder='Password')
+    Confirm_password = st.text_input('',placeholder='Confirm password')
+    Email = st.text_input('',placeholder='E-mail address')
+
+    submitted = st.form_submit_button("Sign Up")
```

Comment:
Can you also add some logic to ensure the user doesn't leave any fields blank? Also if you can add enforcement of strong password guidelines(i.e 8 character minimum, uppercase, lowercase, numbers, and special chars) that would be great.

Reply:
Sounds good, I will deal with that.